src/index.html

```diff
@@ -1,13 +1,440 @@
+                  grid__item
+                  grid__item--tablet-3-6
+                  grid__item--desktop-5-12
+                ">
+                  <picture>
+                    <source srcset="./images/galerey/desktop/6.png"  media="(min-width: 1024px)">
+                    <source srcset="./images/galerey/tablet/6.png" media="(min-width: 744px)">
+                    <source srcset="./images/galerey/mobile/6.png"  media="(max-width: 743px)">
+                    <img
+                      class="type__photo"
+                      src="./images/galerey/mobile/6.png"
+                      alt="Bose"
+                    >
+                  </picture>
+                </div>
+              </div>
+            </div>
+            <a href="#" class="type__link">
+              <div class="type__description">
+                Audio sunglasses
+              </div>
+            </a>
+          </div>
+
+          <a href="#" class="button">
+            View all
+          </a>
+
+        </div>
+      </section>
+
+
+      <section class="page__section about"
+      id="how-to-buy">
+        <div class="container">
+          <div class="grid grid--desktop">
+            <div
+              class="
+                grid__item
+                grid__item--tablet-1-3
+                grid__item--desktop-1-5
+            ">
+              <h2 class="about__title">
+                Why buy direct from Bose
+              </h2>
+            </div>
+            <div
+              class="
+              grid__item
+              grid__item--tablet-4-6
+              grid__item--desktop-7-11
+            ">
+              <ul class="about__list">
+                <li class="about__list-item">Free 2-day shipping and returns</li>
+                <li class="about__list-item">90-day risk-free trial</li>
+                <li class="about__list-item">World class customer service</li>
+                <li class="about__list-item">My Bose account management</li>
+              </ul>
+
+              <div class="about__description">
+                A great product is more than what’s in the box.
+                It’s a promise of premium performance, world-class support,
+                and everything you expect from a trusted brand. It’s just
+                one of many reasons why you’ll shop with confidence on
+                Bose.com.
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-6 contacts__field--last">
+              <form action="#">
+                <input
+                  placeholder="Name"
+                  name="name"
+                  type="text"
+                  class="contacts__field form-field
```

Comment:
```suggestion
                  class="contacts__field form-field
                  required
```

Make input fields required. In your case, it's possible to send a blank form.